src/pytorch_lightning/plugins/precision/apex_amp.py

```diff
@@ -83,6 +84,8 @@ def optimizer_step(
         closure: Callable[[], Any],
         **kwargs: Any,
     ) -> Any:
+        if self._state_dict_loaded:
+            raise RuntimeError("Resuming training with APEX is currently not supported.")
```

Comment:
should we suggest not using apex while configuring the Trainer to avoid this?

Reply:
How would you show this error message during Trainer configuration? What would be the condition?

Reply:
```py
raise RuntimeError("Resuming training with APEX is currently not supported. Please configure your `Trainer(amp_backend=None)` while resuming.")
```

Reply:
oh! my comment might have confused you. I meant let users know how they can update the Trainer configuration to resume without apex :)